spec/models/piece_spec.rb

```diff
@@ -1,6 +1,36 @@
 require 'rails_helper'
 
 RSpec.describe Piece, type: :model do
+  describe 'move_to! method' do
```

Comment:
Here's a trick you can use to DRY up your code even more.

Take a look at `let` => http://betterspecs.org/#let
```Ruby
describe 'move_to! method' do
     let(:game) { FactoryGirl.create(:game) }
     it 'should return error if target piece has same color as current piece' do
       # now you don't have to create a game each time, it's already available :)
       # ..etc...
       current_piece = FactoryGirl.create(:piece, color: 0, game_id: game.id)
```